metabase/metabase
# Fix for empty tooltip shown when save button is disabled

Closes #51035 
### How to verify

1. Open your account and create a new question
2. Select Sample Dataset -> Orders
4. Hover over the Save button when it's enabled - no tooltip should appear


### Demo

https://github.com/user-attachments/assets/c4c1afd2-c62c-4887-9882-ad991b8c43c6


### Checklist
